src/ToolInfo.php

```diff
@@ -73,31 +72,31 @@ public static function isInstalledByComposer()
         static $result;
 
         if (null === $result) {
-            $result = !self::isInstalledAsPhar() && file_exists(self::getScriptDir().self::COMPOSER_INSTALLED_FILE);
+            $result = !self::isInstalledAsPhar() && file_exists(self::getComposerInstalledFile());
```

Comment:
`file_exists()` can be replaced with `null !==` as the file is already checked in `getComposerInstalledFile()`.